src/applications/virtual-agent/components/chatbox/Chatbox.jsx

```diff
@@ -36,21 +41,56 @@ export default function Chatbox(props) {
           VA Virtual Agent (beta)
         </h2>
       </div>
-      <App timeout={props.timeout || ONE_MINUTE} />
+      {!isLoggedIn ? (
+        <ConnectedSignInAlert />
+      ) : (
+        <App timeout={props.timeout || ONE_MINUTE} />
+      )}
     </div>
   );
 }
 
+function SignInAlert({ showLoginModal }) {
```

Comment:
nit: This and `ConnectedSignInAlert` could be moved to its own file, but it's nbd,